schema/schema_test.go

```diff
@@ -255,6 +262,22 @@ func TestSchema_Decode(t *testing.T) {
 			t.Errorf("value:Age want:42 got:%d", t1.Age)
 		}
 	})
+	t.Run("StructWithTags", func(t *testing.T) {
```

Comment:
Could you please add an analogous test to TestSchema_Encode?

Reply:
Oh yes, just missed this. Now the tests are complete